modules/administration-guide/pages/editor-configurations-for-microsoft-visual-studio-code.adoc

```diff
@@ -34,7 +34,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: vscode-editor-configurations
-data: 
+  labels:
+    controller.devfile.io/watch-configmap: "false"
+    controller.devfile.io/mount-to-devworkspace: "false"
+data:
```

Comment:
are these labels actual when the ConfigMap is created in the user's namespace?
or
they are for the `eclipse-che` namespace only?

Reply:
Only when CM is created in eclipse-che namespace

Reply:
The current doc is for the `user's` namespace ConfigMap:
![image](https://github.com/user-attachments/assets/b5039015-69ed-4ed1-a4d1-b3653e5b7d20)

We only have a tip, that the same ConfigMap can be used for the `eclipse-che` namespace.

So, maybe it makes sense to add some additional info about those labels and a difference for `user's` namespace vs `eclipse-che` namespace

Reply:
btw
for the `eclipse-che` namespace we have example of CM here https://eclipse.dev/che/docs/stable/administration-guide/configuring-a-user-namespace/
so - should these 2 labels be part of that example?

Reply:
@RomanNikitenko make sense for me